### DISCUSSION TITLE
Versão informada no pom.xml

### DISCUSSION THREAD

**Discussion by FelipeGutjahr** [wmixvideo/nfe#877] [Geral]
Boa tarde,

No dia 07/03 houve a mudança na URL de consulta em Minas Gerais. Quando isso ocorreu, clientes começaram a apresentar a rejeição 'Endereço do site da UF da consulta por chave de acesso diverge do previsto'. Notei que vocês já resolveram isso, só é necessário atualizar meu sistema com a nova versão. Então eis o problema:

Meu pom.xml estava da seguinte forma:

```
<dependency>
    <groupId>com.github.wmixvideo</groupId>
    <artifactId>nfe</artifactId>
    <version>${latest.release}</version>
</dependency>
```

Teoricamente buscando sempre a última versão, atualmente a 4.0.8, que já está com as novas URLs para Minas Gerais. Porém verifiquei no arquivo DFUnidadeFederativa, e o mesmo continua com as URLs antigas.

Alterei o pom.xml e deixei da seguinte forma:

```
<dependency>
    <groupId>com.github.wmixvideo</groupId>
    <artifactId>nfe</artifactId>
    <version>4.0.8</version>
</dependency>
```

Dessa forma as URLs ficam corretas no arquivo DFUnidadeFederativa. Porém ao tentar gerar o .war do projeto ocorre o seguinte erro:

```
[ERROR] /C:/Users/gutja/Documents/Repository/sispronto-back/master/src/main/java/br/com/controlgas/model/app/Pessoa.java:[11,36] cannot access com.fincatto.documentofiscal.DFModelo
  bad class file: C:\Users\gutja\.m2\repository\com\github\wmixvideo\nfe\4.0.8\nfe-4.0.8.jar(com/fincatto/documentofiscal/DFModelo.class)
    class file has wrong version 55.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
```

Alguém teria uma ideia de como resolver?

**Comment by marcoslombardi** [comment]
Passei pelo mesmo [erro](https://github.com/wmixvideo/nfe/issues/871) consegui resolver configurando o Java 1.8 no projeto nfe e executando o maven install para geração do nfe-X.X.X.jar

**Reply by FelipeGutjahr** [reply]
Fiz o clone do projeto, gerei o nfe-4.0.8.jar e adicionei ao projeto. Porém ao tentar gerar tive o seguinte erro:

```
[ERROR] Supertypes of the following classes cannot be resolved. Please make sure you have the required dependencies in the classpath:
    class com.fincatto.documentofiscal.utils.DFPersister, unresolved supertypes: org.simpleframework.xml.core.Persister
```

Você passou por isso também?
